packages/react/package.json

```diff
@@ -40,8 +40,8 @@
   },
   "peerDependencies": {
     "@parallelmarkets/vanilla": "workspace:^",
-    "react": "^16.8.0 || ^17.0.0",
-    "react-dom": "^16.8.0 || ^17.0.0"
+    "react": "^16.8.0 | ^17.0.0 | ^18.0.0",
+    "react-dom": "^16.8.0 | ^17.0.0 | ^18.0.0"
```

Comment:
I am definitely no expert in `semver`, but is there a reason that we changed from doing a double `||` to a single `|` here?

Quickly trying this out in the console yields
```
> semver.satisfies('16.8.0', "^16.8.0 | ^17.0.0 | ^18.0.0")
false
> semver.satisfies('16.8.0', "^16.8.0 || ^17.0.0 || ^18.0.0")
true
```

Reply:
Good catch!  Yeah, that was a typo on my part 🤦‍♂️   Fixed in 219b52d